README.md

```diff
@@ -17,29 +17,29 @@ Additional access is granted via [snap interfaces](https://snapcraft.io/docs/int
 
 After the installation it's necessary to connect the interfaces:
 
+- [cpu-control](https://snapcraft.io/docs/cpu-control-interface)
+- `etc-default-grub` plug into the [system-files](https://snapcraft.io/docs/system-files-interface) interface;
```

Comment:
```suggestion
- `etc-default-grub` [system-files](https://snapcraft.io/docs/system-files-interface) interface;
```

Reply:
The statement is not correct. An interface is the shared boundary between the plug and slot. An interface plug connects into the interface slot. The `etc-default-grub` system-files interface plug connects to the corresponding slot provided by snapd.